src/dailies.ts

```diff
@@ -495,15 +490,19 @@ function configureVykea() {
       [2, 1],
       [3, 11],
     ]; // excluding 4 and 5 as per bean's suggestion
-    const vykeaProfit = (level: number, cost: number) =>
-      estimatedTurns() * baseMeat * 0.1 * level -
-      (5 * mallPrice($item`VYKEA rail`) +
-        cost * mallPrice($item`VYKEA dowel`) +
-        5 * mallPrice($item`VYKEA plank`) +
-        1 * mallPrice($item`VYKEA instructions`));
-
-    if (vykeas.some(([level, cost]) => vykeaProfit(level, cost) > 0)) {
-      const level = vykeas.sort((a, b) => vykeaProfit(...b) - vykeaProfit(...a))[0][0];
+    const vykeaProfit = (vykea: [number, number]) => {
+      const [level, cost] = vykea;
+      return (
+        estimatedTurns() * baseMeat * 0.1 * level -
+        (5 * mallPrice($item`VYKEA rail`) +
+          cost * mallPrice($item`VYKEA dowel`) +
```

Comment:
okay i realize that this Works :tm: but i would recommend renaming "cost" here. the cost that is being used is the dowelCost, whereas the vykeaCost is the thing actually being calculated. by just naming it cost it reads like there's double counting going on until you reread the whole thing.